simcon_project/simcon_project/urls.py

```diff
@@ -34,19 +34,29 @@
 
     # Stuff students can see
     path('student/', student_view, name="student-view"),
-    path('student/settings/', student_settings_view, name="student-settings-view"),
-    path('student/register/<uidb64>/', student_registration, name="student-registration"),
-    path('student/conversation/', include('conversation_templates.urls.conv_urls'), name='conversation'),
+    path('student/settings/', student_settings_view,
+         name="student-settings-view"),
```

Comment:
Looks like your auto-formatter got a hold of this file lol

Reply:
...how.. i didnt even touch it